sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -228,6 +228,8 @@ def _compile_kernel(self, kernel):
             self.vpp_llvm_input
         ]
         if kernel['extra_args'].strip():
+            # We should remove terminal spaces to prevent v++ searching
+            # non existant file with name ''
             command.extend(kernel['extra_args'].strip().split(' '))
```

Comment:
So it should be `rstrip()` instead.

Reply:
Both of them actually, if the user input a kernel param with a leading space. 
I'll correct the comment.